### PR TITLE
Update readme: how to edit files for beginners

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,44 @@
...
-action publish on every commit. Contributions to the docs are very welcome!
+action publish on every commit. Contributions to the docs, all of which you can find as .md files
+in the [book/src](https://github.com/a-b-street/docs/tree/main/book/src) folder of this repo,
+and which can be edited on GitHub are very welcome!
 
-You can see the deployed docs at <https://a-b-street.github.io/docs>.
+To build the book, download or clone the repo, e.g. using the [`gh` CLI](https://github.com/cli/cli) as follows:
+
+```bash
+gh repo clone a-b-street/docs
```
Comment (michaelkirk) [inline README.md]:
```suggestion
git clone https://github.com/a-b-street/docs
```

gh cli is nice (I use it!), but for docs, I prefer sticking with the most basic tools that most normal folks will have,  (unless of course there's some big advantage to using the fancy tool, but I consider `gh`s utility here pretty marginal). I think it's reasonable to expect that the `gh` "power users" will know how to translate this.

File: book/src/README.md [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+Welcome to the A/B Street website. Here you will find information about the
+project, an overview of the [software](software) products it has enabled, a
+[user guide](user), [technical details](tech) aimed at developers, and lots
+more.
 
+Like any open source project, A/B Street has evolved and will continue to
+evolve, and that applies to the documentation. Any contributions to the website,
+the source code of which can be found at
+[github.com/a-b-street/docs](https://github.com/a-b-street/docs) are very
+welcome. In fact, contributing to the project's documentation can be the one of
```
Comment (michaelkirk) [inline book/src/README.md]:
```suggestion
welcome. In fact, contributing to the project's documentation can be one of
```

File: book/src/README.md [hunk 3]
```diff
@@ -1,10 +1,23 @@
 # Homepage
 
-This documentation is in the process of turning into a proper website.
+Welcome to the A/B Street website. Here you will find information about the
```
Comment (michaelkirk) [inline book/src/README.md:3]: Woohoo! I'd be very happy to see this replaced!

**Review by michaelkirk — COMMENTED** [review]
Nice! Thanks for updating the docs @Robinlovelace! I'm especially excited to see some proper content on the home page.

File: book/book.toml [hunk 4]
```diff
@@ -6,6 +6,10 @@ src = "src"
 title = "A/B Street"
 
 [output.html]
+mathjax-support = true # equations if necessary
```
Comment (dabreegster) [inline book/book.toml]:
```suggestion
```
We can add the mathjax and any other plugin when we first use it.
I haven't looked up what `site-url` does, but if we weren't using it before and things were fine, not sure there's value adding this note

File: book/book.toml [hunk 5]
```diff
@@ -6,6 +6,10 @@ src = "src"
...
+# site-url = "/mdBook/" # not sure what this should be (Robin)
```
Comment (dabreegster) [inline book/book.toml]:
```suggestion
```

File: book/src/README.md (hunk 3 above)
Comment (dabreegster) [reply book/src/README.md:3]: Yesss thank you for picking up after my super brief flurried attempt to redo the site.

**Comment by dabreegster** [comment]
Thank you Robin and Michael! I committed the suggested changes, I'll merge this now

**Pull request merged**